src/lib/style/components/_featured-data-plots.scss

```diff
@@ -1,21 +1,25 @@
 @media (max-width: 800px) {
   .FeaturedDataPlots {
-    display: block;
-    &__card {
-      width: 100%;
-    }
+    &__queryPerCard, &__singleQuery {
+      display: block;
+      &__card {
+        width: 100%;
+      }
+    }    
   }
 }
 @media (min-width: 801px) {
```

Comment:
Flex grid container and item elements depends on the component structure, but same style is applied.